### PR TITLE
Add check status of transaction

### PR DESCRIPTION
**Pull request by dongrie**
Add waiting for the results of the transaction to check the status

### CODE REVIEW

File: cmd/transfer.go [hunk 1]
```diff
@@ -99,7 +107,21 @@ func Transfer(ctx context.Context, rpcAddress string, chainID int64, mnemonic, i
...
 
 	return nil
 }
+
+func waitAndCheckStatus(ctx context.Context, chain *geth.Chain, tx *types.Transaction) error {
+	receipt, err := chain.Client.WaitForReceiptAndGet(ctx, tx)
+	if err != nil {
+		return err
+	}
+	if receipt.Status != 1 {
```
Comment (siburu) [inline cmd/transfer.go:123]: We should avoid using magic numbers. For here, you can use the following.
https://github.com/ethereum/go-ethereum/blob/6a6318b1d23ec20d1a8735cf1852c11726a4ac9a/core/types/receipt.go#L48

Comment (dongrie) [reply cmd/transfer.go]: https://github.com/datachainlab/ethereum-ics20-cli/pull/3/commits/8e980906f51af7880e39be53d99d0229444e05a0

Remove magic number, use const

File: cmd/transfer.go [hunk 2]
```diff
@@ -75,6 +77,9 @@ func Transfer(ctx context.Context, rpcAddress string, chainID int64, mnemonic, i
 		return err
 	}
 	log.Printf("1. token approve success (TxHash: %s)\n", tx.Hash().Hex())
```
Comment (bluele) [inline cmd/transfer.go:79]: We should get this log after `waitAndCheckStatus` is passed.

File: cmd/transfer.go [hunk 3]
```diff
@@ -75,6 +77,9 @@ func Transfer(ctx context.Context, rpcAddress string, chainID int64, mnemonic, i
...
+	if err := waitAndCheckStatus(ctx, chainA, tx); err != nil {
```
Comment (bluele) [inline cmd/transfer.go:80]: nit: Since `chainB` does not exist in the function, can we simply use `chain` instead?

File: chains/geth/chain.go [hunk 4]
```diff
@@ -21,10 +21,10 @@ import (
 
 type Chain struct {
 	chainID        int64
```
Comment (bluele) [inline chains/geth/chain.go:23]: btw, you can get a chain-id via `eth_chainId` method

File: cmd/transfer.go (hunk 2 above)
Comment (dongrie) [reply cmd/transfer.go]: log output is now after the decision process
https://github.com/datachainlab/ethereum-ics20-cli/pull/3/commits/a282bb6171e91448a6026fa7a2ef440e62553f08

File: cmd/transfer.go (hunk 3 above)
Comment (dongrie) [reply cmd/transfer.go]: Rename chainA -> chain
https://github.com/datachainlab/ethereum-ics20-cli/pull/3/commits/a282bb6171e91448a6026fa7a2ef440e62553f08

File: chains/geth/chain.go (hunk 4 above)
Comment (dongrie) [reply chains/geth/chain.go:23]: Sorry, what does this mean?
Where is the method eth_chainId?

Comment (bluele) [reply chains/geth/chain.go:23]: sorry, `eth_chainId ` is a method of json rpc provided by ethereum client

Comment (bluele) [reply chains/geth/chain.go:23]: https://eips.ethereum.org/EIPS/eip-695

Comment (dongrie) [reply chains/geth/chain.go:23]: Ah, so that's how it is. I understand

Comment (dongrie) [reply chains/geth/chain.go:23]: https://github.com/datachainlab/ethereum-ics20-cli/pull/3/commits/febc444353171ca1910dec94a9d8c299ab4165a8

use `eth_chainId` method and removed chainId flag from commands

**Review by bluele — APPROVED** [review]
LGTM 👍

**Pull request merged**